modules/01_macro/singleSectorGr/bounds.gms

```diff
@@ -8,12 +8,17 @@
 
 *nb* lower bounds on CES values
 vm_cons.lo(t,regi)     = 1e-3;
-vm_cesIO.lo(t,regi,in) = 1e-6;
+vm_cesIO.lo(t,regi,in)$( NOT in_industry_dyn37(in) ) = 1e-6;
```

Comment:
The lower bound is in the module, since it is realisation-specific.
https://github.com/0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q/remind/blob/merge__dev__calibrate_industry/modules/37_industry/subsectors/bounds.gms#L58-L61